snap/README.md

```diff
@@ -0,0 +1,43 @@
+# Kata Containers Snap
+
+This directory contains the resources needed to build the Kata Containers
+[snap][1] image.
+
+## Initial setup
+
+*Ubuntu Bionic*
+
+```
+$ sudo apt-get install -y snap
+$ git clone https://github.com/kata-containers/packaging
+```
+
+## Build snap image
+
+Run next command at the root directory of the packaging repository.
+
+```
+$ snapcraft -d
+```
+
+## Install snap (developer)
+
+```
+$ sudo snap install --classic --dangerous kata-containers_[VERSION]_[ARCH].snap
+```
+
+Replace `VERSION` with the current version of Kata Containers and `ARCH` with
```

Comment:
Couldn't we replace `[ARCH]` with `$(arch)`?

Reply:
nop, for example on x86_64 systems $(arch) prints x86_64, but in snapcraft ARCH is amd64

Reply:
Ah - ack. We could point users at [`kata-arch.sh --golang`](https://github.com/kata-containers/tests/blob/master/.ci/kata-arch.sh) but it's a pita that `arch(1)` doesn't have these options itself ;(